src/hooks/use-event-data.js

```diff
@@ -49,6 +49,21 @@ export const sampleEvents = [
     },
 ];
 
+export const removePastEvents = events => {
```

Comment:
Super nit, but I noticed the `React` import on line 1 is no longer used. Should we remove it?

Reply:
Good catch 👍